### ISSUE TITLE
Embedcreator, faulty check

### ISSUE THREAD

**Issue by Roxedus** [flaree/Flare-Cogs#79]
**Describe the bug**
d.py strikes again. Embeds can have a [length of 0](https://github.com/flaree/Flare-Cogs/blob/master/embedcreator/embedcreator.py#L146). You can do this with sending a embed with just a image, which discord handles just fine. 

Using `[p]embed from` on a embed created with another cog yields: 
```json
{
    "image": {
        "url": "https://i.imgur.com/eujcjNp.png",
        "proxy_url": "https://images-ext-2.discordapp.net/external/rjASdEAfp-7dUHCkwO8i5bMlrg-IGC5N6O1OYcCsrIk/https/i.imgur.com/eujcjNp.png",
        "width": 734,
        "height": 129
    },
    "color": 7562986,
    "type": "rich"
}
```

Trying to replace the image with
```json
{
    "image": {
        "url": "https://i.imgur.com/V9vm6q9.png"
    },
    "color": 7562986
}
```
triggers said check, although the embed is in fact valid.

I have no idea on the best way to make this check work(this would then be another PR), short of removing the minimum length, or add an explicit check for attributes that can be alone, such as image. 

![image](https://user-images.githubusercontent.com/7110194/102020234-1b666f80-3d78-11eb-9ffd-83653fb09873.png)

**Comment by flaree** [comment]
Oops, used wrong ID in commit check. Hopefully fixed within b71b01fd882afe5f5c188f8cda264374630162aa. Let me know if that solves it.

**Comment by Roxedus** [comment]
It does, appreciate it
